microsoft-365/business/manage-windows-devices.md

```diff
@@ -32,19 +32,36 @@ To set up your organization's domain-joined devices to benefit from the capabili
+   
+   - It is recommended that the **userPrincipalName** (UPN) attribute for each local user account be configured to match the primary email address that corresponds to the licensed Microsoft 365 user. For example **mary.shelley@contoso.com** rather than **mary@contoso.local**
+   
+   - Finally, if the Active Directory domain ends in a non-routable suffix like **.local** or **.lan**, instead of an internet routable suffix such as **.com** or **.org**, then you will need to adjust the UPN suffix of the local user accounts first as described in [Prepare a non-routable domain for directory synchronization](https://docs.microsoft.com/en-us/office365/enterprise/prepare-a-non-routable-domain-for-directory-synchronization). 
+
+2. **Install and configure Azure AD Connect**: To synchronize your users, groups and contacts from the local Active Directory into Azure Active Directory, run the Directory synchronization wizard from Azure Active Directory Connect as described in [Set up directory synchronization for Office 365](https://support.office.com/article/1b3b5318-6977-42ed-b5c7-96fa74b08846).
     
     > [!NOTE]
     > The steps are exactly the same for Microsoft 365 Business. 
-  
-2. Before you complete step 3 to enable Windows 10 devices to be Hybrid Azure AD joined, you need to make sure that you meet the following prerequisites:
+    
+As you configure your options for Azure AD Connect, we recommend enabling **Password Synchronization** and **Seamless Single Sign-On**,, as well as the **Password writeback** feature, which is also supported in Microsoft 365 Business.
+
+> [!NOTE]
+    > Password writeback has some additional steps to complete beyond the checkmark box in Azure AD Connect. Refer to [How to configure password writeback for Azure AD SSPR](https://docs.microsoft.com/en-us/azure/active-directory/authentication/howto-sspr-writeback). 
+     
+3. **Configure Hybrid Azure AD Join**: Before you proceed to enable Windows 10 devices to become Hybrid Azure AD joined, you need to make sure that you meet the following prerequisites:
 
    - You are running the latest version of Azure AD connect.
 
    - Azure AD connect has synchronized all the computer objects of the devices you want to be hybrid Azure AD joined. If the computer objects belong to specific organizational units (OU), then make sure these OUs are set for synchronization in Azure AD connect as well.
+
+Then, to register existing domain-joined Windows 10 devices as Hybrid Azure AD Joined, follow the step by step instructions in [Configure Hybrid Azure AD Join for managed domains](https://docs.microsoft.com/en-us/azure/active-directory/devices/hybrid-azuread-join-managed-domains#configure-hybrid-azure-ad-join). This will hybrid-enable your existing on-premises Active Directory joined Windows 10 computers and make them cloud ready.
```

Comment:
```suggestion
Then, to register existing domain-joined Windows 10 devices as Hybrid Azure AD Joined, follow the step by step instructions in the [Tutorial: Configure hybrid Azure Active Directory join for managed domains](https://docs.microsoft.com/azure/active-directory/devices/hybrid-azuread-join-managed-domains#configure-hybrid-azure-ad-join). This will hybrid-enable your existing on-premises Active Directory joined Windows 10 computers and make them cloud ready.
```